corfudb-tools/README

```diff
@@ -9,9 +9,9 @@ definitions or proto files to serialize/deserialize the data.
 --------
 Browser
 --------
-The CorfuStore Browser supports the following operations -
-The browser supports read operation which reads and outputs all entries in a
-given table and namespace, or list tables in a given namespace.
+The browser supports read operations which reads and outputs all entries in a
```

Comment:
a bit weird -> 'supports read operations which reads'... perhaps something like 'The browser allows to visualize/inspect corfu tables and metadata... by reading all entries...